pyerrors/correlators.py

```diff
@@ -1084,18 +1081,18 @@ def __str__(self):
 
     __array_priority__ = 10000
 
-    def __eq__(self, y):
+    def __eq__(self, y: Union[Corr, Obs, int]) -> ndarray:
         if isinstance(y, Corr):
             comp = np.asarray(y.content, dtype=object)
         else:
             comp = np.asarray(y)
         return np.asarray(self.content, dtype=object) == comp
 
-    def __add__(self, y):
+    def __add__(self, y: Any) -> "Corr":
```

Comment:
In contrast, here, `y` should be one of the types that don't raise an exception.